release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1306,6 +1306,10 @@ With this update, you can add the `nodeLabels` field in the `SiteConfig` CR to c
 
 With this release, you can set the control plane hardware speed to one of `"Standard"`, `"Slower"`, or the default, `""`, which allows the system to decide which speed to use. This is a Technology Preview feature. For more information, see xref:../scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc#etcd-tuning-parameters_recommended-etcd-practices[Setting tuning parameters for etcd].
 
+[id="ocp-4-14-ztp-siteconfig-deprecation"]
+==== Deprectaion of fields in SiteConfig
+With this update, the `apiVIP` and `ingressVIP` fields in the `SiteConfig` CRD are deprecated in favor of the plural versions `apiVIPs` and `ingressVIPs`.
```

Comment:
```suggestion
With this update, the `apiVIP` and `ingressVIP` fields in the `SiteConfig` CRD are deprecated in favor of the plural forms, `apiVIPs` and `ingressVIPs`.
```

Reply:
We use versions for numbered values of our products. So to avoid confusion, using "plural forms" can be considered.

Reply:
thanks @xenolinux 
Good point.